src/main/java/de/caritas/cob/agencyservice/api/service/AgencyService.java

```diff
@@ -125,8 +130,19 @@ private boolean isTopicFeatureEnabledAndActivatedInRegistration() {
   }
 
   private boolean isTopicFeatureActivatedInRegistration() {
-    // TODO get this flag from the tenant service,  implement within VIC-797
-    return true;
+    RestrictedTenantDTO restrictedTenantDTO;
+    if (multitenancy) {
```

Comment:
please extract the whole if block to a separate method like getRestrictedTenantData()